bin/pycbc_upload_xml_to_gracedb

```diff
@@ -153,22 +198,44 @@ for event in coinc_table:
     xmldoc.childNodes[-1].appendChild(coinc_inspiral_table_curr)
     xmldoc.childNodes[-1].appendChild(coinc_event_map_table_curr)
     xmldoc.childNodes[-1].appendChild(sngl_inspiral_table_curr)
-    ligolw_utils.write_filename(xmldoc, "tmp_coinc_xml_file.xml")
+
+    if args.filename_T050017:
+        ifos = sorted([sngl.ifo for sngl in sngl_inspiral_table_curr])
+        ifos_str = ''.join(ifos)
+        id_str = args.filename_T050017
+        filename_xml = "{}-PyCBC_{}-{:d}-1.xml".format(ifos_str, id_str, time)
+        filename_psd = "{}-PyCBC_{}-{:d}-1_psd.xml.gz".format(ifos_str, id_str, time)
```

Comment:
If you want to stick to the naming convention, I guess `filename_psd` should be something like `H1L1-PYCBC_PSD_FOOBAR-1234-1.xml.gz` instead? And maybe to make things more clear, `filename_xml` could then be `H1L1-PYCBC_TRIGGER_FOOBAR-1234-1.xml.gz` instead.

Reply:
I think the "trigger" bit might not be allowed according to how people downstream are interpreting the naming convention at the moment